### ISSUE TITLE
Lexicon - Datatables column table sort activation only uses aria-label, does not indicate it’s a button

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1595]
**Describe the bug**
Currently the column dataTable header cells have been scripted to switch between three states - sorting from high to low/ sorting from low to high/ remove sorting. Each state has an aria-label (*Column*: Activate to sort/ *Column*: Activate to invert sorting/*Column*: Activate to remove sorting). Across screen readers and browsers support is patchy upon announcing these states: JAWS only announced the aria-label each time the sort button was activated and VO doesn’t announce the label no matter the browser. If the aria-label isn't announced, there is nothing to indicate that the <th> cell will act like button. 

**To Reproduce**
1. Go to Pulsar Lexicon
2. Navigate to Tables > Datatables
3. Inspect the table header cells

**Expected behavior**
The th cells trigger controls, acting like a button so this information should be conveyed to screenreader users by giving the cell the role of button. The current state (sorted/unsorted) is also not currently conveyed again once activated so users may not know if the sorting was successful. This information should be conveyed through the aria-sort attribute or through aria-labelling.

**Screenshots**
<img width="904" alt="Image" src="https://github.com/user-attachments/assets/6074d914-5496-4e2a-94e8-4b8c814e8e3b" />

**Additional context**
Related success criteria:
[4.1.2 Name, Role, Value (A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
[4.1.3 Status Messages (AA)](https://www.w3.org/WAI/WCAG22/Understanding/status-messages)

**Suggested fixes**
As table is based off a 3rd party dataTable library, see if the dataTables can be brought in line with current mark up, eg.  adding an aria-sort attribute (aria-sort="ascending"/aria-sort="descending"/no attribute) to the th cells like in the [current dataTable markup](https://datatables.net/) and placing the arrow controls in a separate span with role button so assistive technology users know to interact with this control.

Additionally if possible, as aria-sort support is patchy, add to the current aria-labels of each column header button state which goes through the cycle (*Column*: Activate to sort/ *Column*: Activate to invert sorting/*Column*: Activate to remove sorting), so instead the three aria-label states announce the current sorting status as well as the upon activation status eg. (*Column*: Currently unsorted, activate to sort/ *Column*: Currently sorted, Activate to invert sorting/*Column*: Currently inverted sort, Activate to remove sorting).
